app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx

```diff
@@ -6,24 +6,23 @@ import Form from "./Form";
 
 import { NOTES_FORM_INITIAL_FORM_VALUES } from "../constants";
 
-const NewNotePane = ({ fetchNotes, showPane, setShowPane }) => {
+const Create = ({ showPane, setShowPane }) => {
   const onClose = () => setShowPane(false);
 
   return (
     <Pane isOpen={showPane} onClose={onClose}>
       <Pane.Header>
         <Typography style="h2" weight="semibold">
-          Create a new note
+          Create a New Note
```

Comment:
Use `Sentence case`. There is not point in Title Casing the words `New` and `Note`.